ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/flow-tracking.md

```diff
@@ -91,6 +91,7 @@ interface Management1
 ```eos
 !
 flow tracking sampled
+   encapsulation ipv4 mpls
```

Comment:
This is being rejected on my test device (apologies for not testing this further before):

```
ceos1(config-flow-tracking-sampled)#encapsulation mpls
% Both ipv4 and ipv6 encapsulations must be set
ceos1(config-flow-tracking-sampled)#encapsulation ipv6 mpls
% Both ipv4 and ipv6 encapsulations must be set
ceos1(config-flow-tracking-sampled)#encapsulation ipv4 mpls
% Both ipv4 and ipv6 encapsulations must be set
```

Reply:
Looks like some combinations are not allowed so we may need to change the model

Reply:
so we can change the keys to ipv4_ipv6_mpls :)

Reply:
and `ipv4_ipv6` probably need to recheck CLI

Reply:
```
s1-spine1(config-flow-tracking-sampled)#encapsulation ipv6
% Both ipv4 and ipv6 encapsulations must be set
s1-spine1(config-flow-tracking-sampled)#encapsulation ipv4
% Both ipv4 and ipv6 encapsulations must be set
s1-spine1(config-flow-tracking-sampled)#encapsulation ipv4 ipv6
s1-spine1(config-flow-tracking-sampled)#
```
So `ipv4 ipv6` and `ipv4 ipv6 mpls' combinations are allowed.
We can have keys - `ipv4_ipv6` and `mpls` ?

Reply:
yes `ipv4_ipv6` and `mpls` would be correct here!